build/configs/rtl8720e/hello/Make.defs

```diff
@@ -0,0 +1,228 @@
+###########################################################################
```

Comment:
Can you please create multiple commits with different changes related to arch, amebad, kernel etc..
It will be easy to review and modular

Reply:
Hi Sangam, thank you for your advice. But it might be hard to separate the commits. In my personal developing branch, my colleagues and I have committed I think around 50 commits these days. It will be hard for me to summarize all commits and   reform them to different changes as suggested.

Reply:
How about doing as @sangam-swami says when we merge new chipset to master with porting completion. How do you think, @sangam-swami ?

Reply:
I agree @sangam-swami 's opinion.
Huge single commit is impossible to review.
I think @fengyunlai could do easily for @sangam-swami 's request.
```
git add apps/
git commit -s 
(1st commit for app)
git add build/
git commit -s
(2nd commit for build)
git add os/arch/
git add os/board/
git commit -s
(3rd commit for bsp)
...
```

Reply:
Hi Mr Chang, yes, I can separate them by folders as you suggested.

Reply:
It's ok because the target branch is not the master. When we merge it into the master, debug log should be cleaned up like removal or converting to xxdbg.

Reply:
Hi Mr Chang and Sangam, I just created 5 commits which is separated by folders, thanks